posts/2019/07/ibis-python-data-analysis-productivity-framework.ipynb

```diff
@@ -0,0 +1,424 @@
Bgni94oZDF1zORzEmEMgKZPJ0dazOi9pYxWus6saopXm0CLXjkYOQUaSs/YJSnR6NXKAI3YpPRZAnib1pyRuVIDFqujp9ZmAHRHSJOk8jJ3iqBDIh3nBGxNprXGnNAhGeCf8wD4cQR3A/hu7vfnk8cor6G8FwCglHpIKXVOKXUuXgSNH4rT0Ikq9QPoMA5n0xoQyZ2s+yJtIwRo5evkFgss7NlsBBSnTVVNcR02Dc/lSf3yn6N23EPgCKhjcpwr9/qpOnr7+sYxWZJkuo7evJ4wp4hCiW5boaXo+wlAzAiImeuc2HAv/3c5ElIfjqBspRfxiKrXUN5rHtT6Ya31A1rrB1519njjh+KQO9QJBtB1v1TNN2AKlSgSMhbcxIi0pmQ8m7dpce7pYWRupEWbnqTVjOdyBQi+syxqMSHAd6602hSuc2WopjxG79wxKdduJck0uI1JFhMKylgOm6GYtNY8+5vteQAvy/1+D4DzxNf0CO8VGbcdAnWBUQtgrDduIqABvnNh4dnE6/ftXKgbIcDJshhRUbdNhsXs6yljAnTeyTvcRtTR23EpznXKUQ11aecxsCJixjqdL2IS1JrCwsQ5Rb2n1Oide/0kWJAlPqHfU2s+MoLPA7hfKXWfUqoH4F0AHim85hEAfz9RD/0IgBta6wvE94qs30mqS4mpHKVQA5BsWjQP7ls1w2lxa/Bsv3UEJnrz61x5GQGtxQZXNZP/HE3jUqV+AI2AnhHlkwBd859V1hLbQTDG9M1nHEZtBrWGBqAfeGPrCGhwKw/CY8lnGY7AOSPQWodKqfcB+CSANoAPa62fUEq9N3n+QwAeBfBOAM8AmAD4B3Xvdf1MgEnlBh0a9j5f0Ao1ADr2zCKMDmGBcaJXKvbcYrTDmC+aW3BboxLw00Q1Q+FyqCRcqvBoOPEO4BOb1BYLAJ2Apt9TxvfUbT5ABrCqIb8wBrcdxmFkWZt92jZIVU1lnCPhnubks3XzmtocD+BdvzUf0BC01o/CbPb5xz6U+1kD+Dnqe30ZZ4MlQ0PdNq5Pmslq6rFyQILnMshiDrE7IUYbdDyfij3To9dBl6oaYsBNxDSeVaTG1PyTnWunRexfFZG06QAPbiRDI13T2j2M4loJa9rEkXQwDe2eaq3JSixekWKMOzbp108tKKO0bAGWYZy6vSKrdWI4AkZ18ZGsLKYaWZ/PIjbp0BAFGgDMBhOEcaPmX0IWN0FjcUyXpQEc+SwPz6a2AiGPSbynHGKPypFw5JPms7YYeDZ9TlHJcurcHxKvX3JPm+bUItKINQ0a6bRb6BDbO/PmKS0IYn33Xdo6pdY6mTH50NAt7Qj65EiTuWn5jrTsYmgYl6ujB5o5gnnaj96zIyCqMVhjMjI36j210BD1YBoAjSoPjtTPvI5+/ZSIEDAZEYl3IBbTAXRJtowjaJj7DLIUoNeRzIhcjv3btIwgJM0ngE6WS8jitSNIjMrITxcxnSxmEJuciNC8p37hshQu1OiNMaZ9HZXc4ilcaLUZnI3Afo6mMe1nII/ZtGjtmJ45ElbmSqyAnwZ050K9fpYjIM59GzFTP2ufmmUSuRyAIUlmfE/kdcqUpAJrR5AaddOasyYDlYSL6aoZ4hcnIYvpY9LxfCo0RlcN0XkHDoQHNOOkHKkf9cCblNjzPKcoR59mY3JgUdqYVCXaLKDfU+pGyGniCJh76rNaG+BIkmNS9b8dE+AELBR1382pLP6eNSpZfBjpISd6Iy+GIIJSRKkfERri9Jqxr6NVQnI5Ar/Q0IC4wDiRVq/dQotw4A1n0dq/TYYwiVkGBxrhRMSA3yyTSsBzoSEKhJtyOcS1T5Xkms6rzHnq8Z5yOgBYu6UdAfXINi65QyJ2GRshVTkxC01BDUWN0Gm3SG0GODJXIIFxqJGWZ6kjV+FiPkcDR7AI0eu0Gk+8AxJJMmEz4MJtlF5TVjXju0hvynAuVrHk8/qtasr/PG2+/pTL4WRuRH6Qqu7K2kE0QGMc1ZCgsviWdgQbPfqmxY2KmlJErnzSvqfOqCdpWaNkRBwlEsCBhnjRO6VtLrcK1H6O2jEZqhmAFr1zFi1go9f6e8pRzaRjUhUuTBijKcucJJkrXeHUrJriFFOaMZuvPz3xjRqwEBGGSRCyODeAwTsR7mla70M8AhO41R0BQTWktTbROwN7BAjYe8BL40ljMuAmO27TxJ0woSHKRriIYoSxZmUZQLNzpbZtyI9JSbmpCg87LpksJQsQmvFsCTQyJ2SucwbvQoUwJ/OQrKO3n9U33Mb6npgcQbPMm8c7ABSOwHTz7RHqkoBEMUk8DbYsLgAAIABJREFU5wC45R1B82SYhzG05kVvAEHqycTIARr2TCVgAUPuNRWUWUdA3Qx5i5bpCAiqKQ6eCzQvsAk3I+i1Gz/nnE0WE+4pUzVDda6cYspMPlrfb2jCdK6UHj7cTZtSmyJxLgDtnlJalgCZAIGSEQyYzpXSa8raLe0IKHhuumi5ahTCZsBXDTVPXM6mRbn+TEfPqFilKjy40BihjsI3scnB3QEeNORTicVpMZD/25TMlVxQRibgeRAmBcaR3FNqluUbxuFV6h8OGkCFcK3d8o6gKZVjp9xUYleiefc8GUzb4IboLVnUI1b06o/YAjj3lJ5yp1WwBGKTtWkRUu7DqM3I5qk/NYqFRbnQUHOWGWJI6N1kzbQDoWVZPpVoXN6Bsk7jWCdzikgWM6AhbsCyVg0lRjmli9PVDwC58Ri3bYV9T50dBlmc9eOnf9YgihHVOFfOqVf519U5GK7Uj6p5px5cb42CPbOJTQ7c5pF34sKi1I6+kyDCsM/JCJrrSPgcSXP/rmkKt9Fx9/z7yiyt1Cd+zi6xHQZHfAHQxQLWbm1HQPDg3PSQrnumq4b6hEVrnqdXQAO2f3z9mGMmNJQerVgD40iKf4D674kr9bPX09Q/n1MBDdAiLS5Z3E8y1zrVFN+5NEOYXEmmUgpDwpyaCpxrI4zD5QgocBNz7Y8I8tksG
```

Comment:
these links don't work unless you're a Twitter user and logged in it seems. either way, for the funding an email address seems better (or leave it out)